doc/releases_and_migration.rst

```diff
@@ -23,6 +23,7 @@ See information about the latest release notes and the available migration guide
    :glob:
    :caption: Migration guides:
 
+   releases_and_migration/migration_guide_addon_v010.rst
    releases_and_migration/migration_guide_v280.rst
    releases_and_migration/migration_guide_v260.rst
-   releases_and_migration/migration_guide_addon_v010.rst
+
```

Comment:
CI failure: "Please remove blank lines at end of 'doc/releases_and_migration.rst'"

Reply:
Yes, was just waiting for any other suggestions before fixing the compliance :)